### PR TITLE
Add March post :)

### PR DESCRIPTION
**Pull request by d-m-u**
🎇

### CODE REVIEW

File: site/_posts/2018-04-04-quandries.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: quandries
```
Comment (cybette) [inline site/_posts/2018-04-04-quandries.md]: Is the misspelling intentional? I never know with you @d-m-u ;)

Also, I'd still prefer if the blog post title is prefaced with "Last Month in ManageIQ". Thanks!

File: site/_posts/2018-04-04-quandries.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+author: d-m-u
+date: 2018-04-04
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Since its founding in 2018, Last Month In MIQ has been the absolute golden standard for journalistic integrity, never afraid to grab truth by the scruff of the neck and shake it up a little. Of course, that being said, we do ever so occasionally take a break from our regularly scheduled program to bring you insights into the minds of our crack team here at MIQ. I know you look forward to such rare occasions with baited breath. Best not to think about what it's baited with.
```
Comment (cybette) [inline site/_posts/2018-04-04-quandries.md]: Isn't it `bated`? ok nvm... :stuck_out_tongue:

Comment (d-m-u) [reply site/_posts/2018-04-04-quandries.md]: This one isn't wrong!

Comment (cybette) [reply site/_posts/2018-04-04-quandries.md]: I reali~z~sed :wink:

**Pull request merged**